src/main/java/com/revature/controllers/SurveyController.java

```diff
@@ -29,6 +29,16 @@
 	public List<Survey> findAll() {
 		return surveyService.findAllOrderByDateCreatedDesc();
 	}
+	
+	@GetMapping("template")
```

Comment:
should be /surveys/template/{isTemplate}
where true or false is passed in

Reply:
done